src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -159,15 +159,12 @@ public RenameRewriter(RenameRewriterParameters parameters)
 
             var isInConflictLambdaBody = false;
             var lambdas = node.GetAncestorsOrThis(n => n is SimpleLambdaExpressionSyntax or ParenthesizedLambdaExpressionSyntax);
-            if (lambdas.Count() != 0)
+            foreach (var lambda in lambdas)
             {
-                foreach (var lambda in lambdas)
+                if (_conflictLocations.Any(cf => cf.Contains(lambda.Span)))
```

Comment:
no 'any' that takes an arg?

Reply:
There isn't one available, and the only current extension methods we have on ImmutableHashSet are in the compiler, and I didn't want to change that in this PR.